docs/source/install/system_requirements.rst

```diff
@@ -30,3 +30,5 @@ for the best experience while testing or deploying |st2|:
 |  * 10GB storage                      | * 40GB storage                    |
 |  * Recommended EC2: **t2.medium**    | * Recommended EC2: **m4.xlarge**  |
 +--------------------------------------+-----------------------------------+
+
+|st2| requires the following TCP ports to be open for successful installation and operation: nginx (80, 443), mongodb (27017), rabbitmq (4369, 5672, 25672), and st2 (9100-9102).
```

Comment:
> the following TCP ports ~to be open~

I think it's a bit confusing, because it can be understood as opening ports in firewall.
And we don't need to expose any ports to public net, except of web.

So IMO something like:
> |st2| uses the following TCP ports [...]

would be more appropriate.